### PR TITLE
Add `n_workers` to output of `status`

### PR DESCRIPTION
**Pull request by abetomo**

### CODE REVIEW

File: lib/proc.c [hunk 1]
```diff
@@ -722,6 +722,8 @@ proc_status(grn_ctx *ctx, int nargs, grn_obj **args, grn_user_data *user_data)
       GRN_OUTPUT_UINT64(0);
     }
   }
+  GRN_OUTPUT_CSTR("n_workers");
+  GRN_OUTPUT_INT32(grn_ctx_get_n_workers(ctx));
```
Comment (kou) [inline lib/proc.c:726]: We must increment `n_elements` too.

It's better that we append a new element to the last. Theoretically adding a new element into a map is backward compatible, but I'm not sure that all clients (for all output formats) are implemented as expected.

Comment (abetomo) [reply lib/proc.c]: I didn't notice `n_elements`. Thanks for the info.
I also moved `n_workers` to last.

**Comment by kou** [comment]
Could you update https://groonga.org/docs/reference/commands/status.html#return-value too?

**Comment by abetomo** [comment]
I have updated the return value in the documentation.

**Review by kou — COMMENTED** [review]
Ah, we want to add `default_n_workers` not `n_workers`.

**Comment by kou** [comment]
We can add `n_workers` too but `default_n_workers` is required.
(Like `command_version` and `default_command_version`.)

**Comment by kou** [comment]
One of this use cases is that we use it to confirm whether the `GRN_N_WORKERS_DEFAULT` environment variable is passed correctly or not.

File: doc/source/reference/commands/status.rst
Comment (kou) [inline doc/source/reference/commands/status.rst:1]: Could you also update the example output at https://groonga.org/docs/reference/commands/status.html#return-value ?

**Comment by abetomo** [comment]
I added  `default_n_workers`.

File: doc/source/reference/commands/status.rst [hunk 2]
```diff
@@ -72,7 +72,10 @@ The command returns the current status as an object::
       "uptime": UPTIME,
       "version": VERSION,
       "features": FEATURES,
-      "apache_arrow": APACHE_ARROW_INFORMATION
+      "apache_arrow": APACHE_ARROW_INFORMATION,
+      "n_workers": N_WORKERS,
+      "default_n_workers": DEFAULT_N_WORKERS
+
```
Comment (kou) [inline doc/source/reference/commands/status.rst:78]:
```suggestion
```

File: doc/source/reference/commands/status.rst [hunk 3]
```diff
@@ -188,3 +191,11 @@ values:
              "version_patch": 0,
              "version": "2.0.0"
           }
+
+   * - ``n_workers``
+     - The value of ``n_workers`` set in this context.
+     - ``0``
+
+   * - ``default_n_workers``
+     - The default ``n_workers`` value of the Groonga process.
```
Comment (kou) [inline doc/source/reference/commands/status.rst:200]: TODO: Add a document for the common `n_workers` parameter to https://github.com/groonga/groonga/tree/main/doc/source/reference/command like https://github.com/groonga/groonga/blob/main/doc/source/reference/command/command_version.rst (but in English) and link from here. The document must describe the `GRN_N_WORKERS_DEFAULT` environment variable too.

**Pull request merged**